src/components/Reports/StaffExpenseReport/StaffExpenseReport.tsx

```diff
@@ -0,0 +1,587 @@
+  Typography,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { DateTime } from 'luxon';
+import { useSnackbar } from 'notistack';
+import { useTranslation } from 'react-i18next';
+import {
+  HeaderTypeEnum,
+  MultiPageHeader,
+} from 'src/components/Shared/MultiPageLayout/MultiPageHeader';
+import {
+  BreakdownByMonth,
+  Fund,
+  SubCategory,
+  TransactionCategory,
+} from 'src/graphql/types.generated';
+import { useLocale } from 'src/hooks/useLocale';
+import theme from 'src/theme';
+import { BalanceCard } from './BalanceCard/BalanceCard';
+import { DownloadButtonGroup } from './DownloadButtonGroup/DownloadButtonGroup';
+import { useReportsStaffExpensesQuery } from './GetStaffExpense.generated';
+import { TableType } from './Helpers/StaffReportEnum';
+import {
+  dateRangeToString,
+  getFormattedDateString,
+} from './Helpers/formatDate';
+import { Filters, SettingsDialog } from './SettingsDialog/SettingsDialog';
+import { EmptyReportTable } from './Tables/EmptyReportTable';
+import { PrintTables } from './Tables/PrintTables';
+import { StaffReportTable } from './Tables/StaffReportTable';
+import { filterTransactions } from './filterTransactions';
+
+const ScreenOnly = styled(Box)({
+  '@media print': {
+    display: 'none',
+  },
+});
+
+const PrintOnly = styled(Box)({
+  display: 'none',
+  '@media print': {
+    display: 'block',
+  },
+});
+
+const StyledHeaderBox = styled(Box)({
+  display: 'flex',
+  alignItems: 'center',
+  gap: theme.spacing(2),
+  justifyContent: 'space-between',
+});
+
+const StyledPrintButton = styled(Button)({
+  border: '1px solid',
+  borderRadius: theme.spacing(1),
+  marginLeft: theme.spacing(2),
+  paddingLeft: theme.spacing(2),
+  paddingRight: theme.spacing(2),
+  paddingTop: theme.spacing(1),
+  paddingBottom: theme.spacing(1),
+});
+
+const StyledTimeNavBox = styled(Box)({
+  display: 'flex',
+  margin: 0,
+  gap: theme.spacing(2),
+});
+
+const StyledFilterButton = styled(Button)({
+  color: theme.palette.cruGrayDark.main,
+  borderColor: theme.palette.cruGrayDark.main,
+  '&:hover': {
+    backgroundColor: theme.palette.cruGrayLight.main,
+    borderColor: theme.palette.cruGrayDark.main,
+  },
+});
+
+export interface Transaction extends BreakdownByMonth {
+  fundType: string;
+  category: TransactionCategory['category'];
+  subcategory?: SubCategory['subCategory'];
+}
+
+interface StaffExpenseReportProps {
+  accountId: string;
+  isNavListOpen: boolean;
+  onNavListToggle: () => void;
+  title: string;
+  time: DateTime;
+  setTime: (time: DateTime) => void;
+}
+
+export const StaffExpenseReport: React.FC<StaffExpenseReportProps> = ({
+  isNavListOpen,
+  onNavListToggle,
+  title,
+  time,
+  setTime,
+}) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const { enqueueSnackbar } = useSnackbar();
+  const [isSettingsOpen, setIsSettingsOpen] = useState(false);
+  const [filters, setFilters] = useState<Filters | null | undefined>(null);
```

Comment:
Can we avoid having to handle two values for missing filters?

Also, I'm wondering if it would simplify things by having `{}` represent no filters since the fields are optional? I'm not saying that's a good idea, I'm just wondering.

```suggestion
  const [filters, setFilters] = useState<Filters | null>(null);
```